client/src/app/shared/components/logo/logo.component.ts

```diff
@@ -96,6 +103,24 @@ export class LogoComponent implements OnInit {
         }
     }
 
+    /**
+     * Check if the user uses a dark theme or a 'bright' theme.
+     * In relation to the theme this will return the corresponding imagepath.
+     *
+     * @returns path of the image corresponding to the chosen theme.
+     */
+    protected getImagePathRelatedToTheme(): string {
+        let path = '';
+        this.configService.get('openslides_theme').subscribe(newTheme => {
```

Comment:
either you have to use `instant`, convert this `get`-call to a promise, or return an observable. But this is prone to give false values when the config-observalbe takes too long (e.g. needs a second). An empty string will always be returned.

You can test this with `.pipe(delay(1000))`.

Reply:
I would prefer making it async with `.pipe(take(1)).toPromise()`.